src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -80,6 +80,8 @@ public static Status GetStatus(this Activity activity)
         /// </summary>
         /// <param name="activity">Activity instance.</param>
         /// <param name="kind">Activity execution kind.</param>
+        /// <remarks>This is required for instrumentations which are dealing with "legacy" activities which has
+        /// the default 'internal' kind, and must be changed to right one.</remarks>
```

Comment:
I'm thinking people may not know what we mean by "legacy".

Suggestion:
"This extension method should only be used on Activity instances that were created via older means that predate ActivitySource. For example, DiagnosticListener.StartActivity"

Reply:
done!